modules/core/shared/src/main/scala/gem/config/DynamicConfig.scala

```diff
@@ -186,4 +191,34 @@ object DynamicConfig {
       GmosSouth(GmosCommonDynamicConfig.Default, None, None, None)
   }
 
+  import GnirsConfig.GnirsFpu
+
+  final case class Gnirs(
+    camera:               GnirsCamera,
+    decker:               GnirsDecker,
+    disperser:            GnirsDisperser,
+    disperserOrder:       GnirsDisperserOrder,
+    exposureTime:         Duration,
+    filter:               GnirsFilter,
+    fpu:                  GnirsFpu,
+    prism:                GnirsPrism,
+    readMode:             GnirsReadMode,
+    wavelengthSuggestion: GnirsWavelengthSuggestion
+  ) extends DynamicConfig.Impl(Instrument.Gnirs)
+
+  object Gnirs {
+    val Default: Gnirs = Gnirs(
+      GnirsCamera.ShortBlue,
+      GnirsDecker.Acquisition,
+      GnirsDisperser.D32,
+      GnirsDisperserOrder.Three,
+      java.time.Duration.ZERO,
+      GnirsFilter.Order5,
+      GnirsFpu.Slit(GnirsFpuSlit.LongSlit4),
+      GnirsPrism.Mirror,
+      GnirsReadMode.Bright,
+      GnirsWavelengthSuggestion.One
```

Comment:
Is this a field of GNIRS?  I would guess that the actual observing wavelength is what we want to store and that the "suggestions" are for the UI?

Reply:
`WavelengthSuggestion` only shows up because of renaming it from `WaveLengthSuggestion`.

I can remember when doing the enumerations that I had the exact same inclination of not considering this parameter since it sounds like UI code only, but the fact is that this suggestion is used in different instances (through getters and setters) in code that is not related to UI.

It's still not clear to me how is it being used in the current OCS, but let me investigate it further. I should, at least, write a comment of why the suggestion is an enumeration.

Reply:
I can't figure out now how this parameter is is being used in OCS, not even in the UI. I'll remove it in gem. We can always add it back if the need arises.